tests/integration/log-proxy-tester/lib/charms/observability_libs/v0/juju_topology.py

```diff
@@ -0,0 +1,288 @@
+# Copyright 2022 Canonical Ltd.
```

Comment:
Was this committed by mistake? Should probably be copied into as part of the test.

Reply:
No, it was not. Since this lib isn't actually part of Loki, and a scenario of "this lib has changes in this patch, so make sure it's copied in for the tests" isn't a common use case, updates as-needed (when/if JujuTopology updates) would be expected/normal.

This also provides an additional validation point for when changes in `loki_push_api` may break/require updates to `JujuTopology` in users, which would get masked if we copied it.